utils/query_builder.py

```diff
@@ -69,6 +85,19 @@ def add_sql_not_in_condition(
             self.where_conditions.append(f"c.{attribute} NOT IN {ids_values}")
         return self
 
+    def add_sql_non_existent_attribute_condition(self, attribute: str):
+        condition = f"""
+        (NOT IS_DEFINED(c.{attribute}) OR c.{attribute} = null)
+        """
+        self.where_conditions.append(condition)
+        return self
+
+    def add_sql_ignore_id_condition(self, id: str = None):
```

Comment:
Mmm I wonder if you really need this method for all the entities or if you just need for a single entity. In code looks as generic but its use seems very specific to me. Please double check if this method only is going to be used for one entity.

Reply:
Resolved with the changes agreed at the meeting.